### DISCUSSION TITLE
Absolutely in love with the flower crown!

### DISCUSSION THREAD

**Discussion by lunulae** [Monika-After-Story/MonikaModDev#8884] [appreciation]
Happy New Year to the MAS team!
I wanted to pop in with my own thanks to the artists for such beautiful holiday outfits. You've all done such an amazing job with this year's events.

I was wondering if there might be any future plans to let us mix and match outfit pieces with other outfits--I'd really like to be able to use that beautiful flower crown as an accessory with other clothes.
I know a bracelet/necklace framework is still in the works, so I understand if things like headpieces will be a ways off. <3

Thank you again for all the work you do, it's been an amazing year and I am excitedly looking forward to your future plans.

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
